### PR TITLE
add prints to activating precompiles

### PR DESCRIPTION
**Pull request by ceyonur**
adds logs to activating/deactivating upgrades

### CODE REVIEW

File: params/precompile_config.go [hunk 1]
```diff
@@ -320,13 +342,18 @@ func (c *ChainConfig) CheckConfigurePrecompiles(parentTimestamp *big.Int, blockC
...
 				// Calling Finalise here effectively commits Suicide call and wipes the contract state.
 				// This enables re-configuration of the same contract state in the same block.
 				// Without an immediate Finalise call after the Suicide, a reconfigured precompiled state can be wiped out
 				// since Suicide will be committed after the reconfiguration.
 				statedb.Finalise(true)
 			} else {
+				// TODO: very janky solution, instead implement stringer in the interface
+				// currently skipping this so we don't break current precompiles
+				configJSON, _ := json.Marshal(config)
+				log.Info("Activating new precompile", "name", key, "config", string(configJSON))
```
Comment (aaronbuchwald) [inline params/precompile_config.go:356]: I think it would be cleaner to simply add a log within the `Configure` function of each of the precompiles.

Comment (ceyonur) [reply params/precompile_config.go:356]: I'd like to keep them in a more common place, also we cannot log from `Configure` when they're disabled. Instead we should implement `fmt.Stringer`.

Comment (ceyonur) [reply params/precompile_config.go:356]: I implemented the stringer interface and simply returned json representation of configs

**Review by darioush — APPROVED** [review]

File: params/precompile_config.go (hunk 1 above)
Comment (aaronbuchwald) [reply params/precompile_config.go]: Ah good point, you're 100% right

**Review by aaronbuchwald — APPROVED** [review]

**Comment by aaronbuchwald** [comment]
Resolves https://github.com/ava-labs/subnet-evm/issues/199

**Pull request merged**